scripts/verify.sh

```diff
@@ -30,7 +30,7 @@ verify_if_multiple_pods_exists() {
     echo "Verifying if multiple pods exists"
     count=0
     MULTIPLE_POD_EXISTS=false
-    while [ "$count" -le 3 ]; do
+    while [ "$count" -le 10 ]; do
```

Comment:
Why are we increasing retry count to 9 times from 3 any particular reason ?

Reply:
When testing macos it happened that the pod for debugging is still terminating after 3 retries, so the test fail. 
See the [run](https://github.com/cxznmhdcxz/Bridge-To-Kubernetes/actions/runs/5737324305/job/15548735505)
Then manually I check pods a bit later and found it's terminated, so I raised the count.